frontend/src/forms/MonsterGallery.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+export default function MonsterGallery() {
+  const amount = 2;
+  let images = [];
+  let currentImage = 0;
+
+  function initImages() {
+    for (let i = 0; i < amount; i++) {
+      images[i] = 'monster' + i + '.jpg';
+    }
+  }
+  initImages();
```

Comment:
Ways too complicated and imperative, try this

```suggestion
  const images = ['monster1.jpg','monster2.jpg']
  const [currentImage,setCurrentImage] = useState(0)
```